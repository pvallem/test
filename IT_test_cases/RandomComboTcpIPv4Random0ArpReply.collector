bidir_flow_info {
  key {
    src_address: "\0135\241\203"
    dst_address: "\315\333\330\322"
    src_port: 14322
    dst_port: 34322
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 10
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "h\256\320D"
    dst_address: "\271\261z\313"
    src_port: 21818
    dst_port: 51052
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "6\3046\001"
    dst_address: "<@8/"
    src_port: 33740
    dst_port: 16306
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\007"
    dst_address: "\001\002\003\010"
    proto: 250
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000010
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
