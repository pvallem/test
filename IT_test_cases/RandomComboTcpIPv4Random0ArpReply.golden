flow_info {
  key {
    src_address: "<@8/"
    dst_address: "6\3046\001"
    src_port: 16306
    dst_port: 33740
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\001\002\003\007"
    dst_address: "\001\002\003\010"
    proto: 250
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  start_time: 1388563200000010
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "\315\333\330\322"
    dst_address: "\0135\241\203"
    src_port: 34322
    dst_port: 14322
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 10
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\271\261z\313"
    dst_address: "h\256\320D"
    src_port: 51052
    dst_port: 21818
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
